/***************************************
*                                      *
*            finally clause            *
*                                      *
***************************************/

try{
    let a=0;
    console.log(program)
    console.log("Program run successfully");
    return  // 
}
catch(err){
    console.log("This is an error")
}
finally{
    console.log("hey!this is finally")
    //close the file
    //exit the loop
 }

 //finally will run whether the try statement is executed fully or not.